code/game/objects/structures/extinguisher.dm

```diff
@@ -59,6 +59,13 @@
 	attack_hand(user)
 	return
 
+/obj/structure/extinguisher_cabinet/AltClick(var/mob/user)
+	..()
```

Comment:
IIRC you only need to call parent if your action fails. That is, if `user.incapacitated() || !Adjacent(user)`, then call parent. Otherwise you'd also be updating the alt-click atom menu thingy and that's annoying.

Reply:
I'll take your word for it. Looks like it's what's done elsewhere too.